server/app/main.py

```diff
@@ -22,19 +24,29 @@ def load_environment():
 
 # Load the environment variables
 load_environment()
-
 # Create the FastAPI app
+
 app = FastAPI()
 
-# Add CORS middleware to the FastAPI app
+
+# Add CORS middleware
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=["http://localhost:3000"],  # Adjust the origin to match your frontend
```

Comment:
pull latest changes, these are already added from earlier PRs

Reply:
Hey Daksha, just wanted to know how was this resolved?